watertap/property_models/unit_specific/anaerobic_digestion/tests/test_modified_adm1_reaction.py

```diff
@@ -940,6 +940,9 @@ def model(self):
         m.fs.unit.liquid_phase.properties_in[0].TSS
         m.fs.unit.liquid_phase.properties_out[0].TSS
 
+        iscale.calculate_scaling_factors(m)
+        iscale.set_scaling_factor(m.fs.unit.liquid_phase.heat, 1e-6)
```

Comment:
WHy did you need this change?

Reply:
This change is not needed. It may have been necessary at some point while I was working on this PR, but I'll undo it